package/MDAnalysis/core/groups.py

```diff
@@ -790,6 +803,7 @@ def center(self, weights, pbc=False, compound='group', unwrap=False):
             compounds
         .. versionchanged:: 0.20.0 Added `unwrap` parameter
         .. versionchanged:: 1.0.0 Removed flags affecting default behaviour
+        .. versionchanged:: 2.0.0 Renamed `pbc` kwarg to `wrap`
```

Comment:
add deprecation note here, too, — `pbc` is still accepted as a synonym for `wrap` but will be removed in 3.0